pybtp/btp.py

```diff
@@ -166,6 +166,8 @@
     "write": (defs.BTP_SERVICE_ID_GATT, defs.GATT_WRITE, CONTROLLER_INDEX),
     "write_long": (defs.BTP_SERVICE_ID_GATT, defs.GATT_WRITE_LONG,
                    CONTROLLER_INDEX),
+    "write_reliable": (defs.BTP_SERVICE_ID_GATT, defs.GATT_WRITE_RELIABLE,
```

Comment:
BTP spec shall be updated as well

Reply:
There is a Reliable Write command in the BTP spec already.

Reply:
OK :)